tools/bazel-container

```diff
@@ -49,6 +49,7 @@ main() {
 
   local -a docker_options=(
     '-it'
+    '--rm'
```

Comment:
This was added before here: https://github.com/world-federation-of-advertisers/experimental/pull/87
but got lost

Reply:
Yeah I had added that locally myself.  Good call.

Reply:
Anything else you added locally that you think is worth adding here?